README.md

```diff
@@ -1,22 +1,29 @@
-# Requirements
+## Requirements
 - Docker
 - Docker-compose
 
-# Getting Started
-- docker-compose build
-- docker-compose run web bin/rails db:setup
-- docker-compose up
+## Getting Started
+- docker compose build
+- docker compose run web bin/rails db:setup
+- docker compose up
 - visit http://localhost:3000/
 
-# Run tests
-- docker-compose exec web bundle exec rspec
+## Run tests
+- docker compose exec -e RAILS_ENV=test web bundle exec rspec
```

Comment:
I played with it locally, I think it would be nice if we don't need to say env=test every time we run rspec WDYT?

Reply:
Done here: [69fa425](https://github.com/espoo-dev/rails_boilerplate/pull/124/commits/69fa425d698dc6bb4092757eb8fea1e61092df8a)